conda/dgl/meta.yaml

```diff
@@ -19,6 +19,7 @@ requirements:
     - scipy
     - networkx
     - requests
+    - tqdm
```

Comment:
Is this necessary? Because tqdm may have unexpected bugs based on user environment, such as terminal without certain functions or writing to logs.